sheetshuttle/util.py

```diff
@@ -26,6 +26,17 @@ def gh_token_exists() -> bool:
     return False
 
 
+def extract_sheet_id(url: str) -> str:
+    """Extract a sheet ID from a Google Sheets URL."""
+    url_segments = url.split('/')
+    collect = False
+    for segment in url_segments:
+        if collect:
+            return segment
+        if segment == "d":
+            collect = True
```

Comment:
you might be able to further simplify the logic to something like this. We're basically saying, after we split with `/`, we want the item after the `d` right?

so maybe try this instead:

```python
url_segments = url.split('/')
for i in range(len(url_segments)):
    if url_segments[i] == "d":
        return url_segments[i+1]
```
Obviously, this has the problem of causing and index out of bounds exception if you pass it an input like this, but you can probably handle that case.

Input like this can cause a problem:
`some_stuff/d/` note that in this case, there wouldn't be an item after the `d`

Reply:
Your code suggestion has been implemented, and exception handling has been added--thanks!